dockers/docker-dhcp-relay/start.sh

```diff
@@ -18,4 +18,13 @@ if [ $(supervisorctl status | grep -c "^isc-dhcp-relay:") -gt 0 ]; then
 
     # Start all DHCP relay agent(s)
     supervisorctl start isc-dhcp-relay:*
+
+    # Start all DHCP Monitor daemon(s)
+    supervisorctl start dhcpmon:*
+fi
+
+# If our supervisor config has entries in the "dhcpmon" group...
+if [ $(supervisorctl status | grep -c "^dhcpmon:") -gt 0 ]; then
+    # Start all DHCP Monitor daemon(s)
+    supervisorctl start dhcpmon:*
 fi
```

Comment:
This block is redundant with lines 22-23. Pick one method. I think lines 22-23 are sufficient, because if there are no DHCP relay agents, there will be no need to monitor, and vice-versa.

Reply:
I thought about having them independent as if there is no dhcpmon in the supervisord config, there would be no need to launch a binary that is has been taken out. On the other hand, you are right if there is no dhcp relay, there is no need to monitor dhcp packets.

Reply:
I'm fine with either approach. Choose one and remove the other.